<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoConfigWidget</class>
 <widget class="QWidget" name="VideoConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_title">
       <property name="text">
        <string>视频常用配置</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../resources.qrc">:/img/Resources/ui/document.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>&lt;a href=&quot;https://www.volcengine.com/docs/6348/70122&quot;&gt;参考文档</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_u" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>240</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="UserWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="UserWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>240</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_14">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>6</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_t1">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>进出房间设置</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">QLabel{color:#42464E; background-color:transparent;}</string>
       </property>
       <property name="text">
        <string>房间id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_room">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">QLabel{color:#42464E; background-color:transparent;}</string>
       </property>
       <property name="text">
        <string>用户id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_uid">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_join">
       <property name="minimumSize">
        <size>
         <width>112</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>112</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>进房</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>本端视频设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">QLabel{color:#42464E; background-color:transparent;}</string>
          </property>
          <property name="text">
           <string>采集分辨率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_c_w">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>640</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_c_h">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>480</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">QLabel{color:#42464E; background-color:transparent;}</string>
          </property>
          <property name="text">
           <string>采集帧率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_c_fps">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">QLabel{color:#42464E; background-color:transparent;}</string>
          </property>
          <property name="text">
           <string>采集偏好</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_capture_preference">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Auto(默认，自动设置采集参数)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Manual(手动设置采集参数，包括采集分辨率、帧率)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Performance(采集参数与编码参数一致)</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_fix_capture">
          <property name="minimumSize">
           <size>
            <width>118</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>修改采集参数</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="1" column="4">
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">QLabel{color:#42464E; background-color:transparent;}</string>
          </property>
          <property name="text">
           <string>kbps</string>
          </property>
         </widget>
        </item>
        <item row="0" column="11" rowspan="2">
         <widget class="QPushButton" name="btn_fix_encoder">
          <property name="minimumSize">
           <size>
            <width>118</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>修改编码参数</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="spinBox_e_bps_max">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>500</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="spinBox_e_h">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>480</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_e_bps_min">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">QLabel{color:#42464E; background-color:transparent;}</string>
          </property>
          <property name="text">
           <string>编码分辨率</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">QLabel{color:#42464E; background-color:transparent;}</string>
          </property>
          <property name="text">
           <string>编码码率</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_19">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>~</string>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">QLabel{color:#42464E; background-color:transparent;}</string>
          </property>
          <property name="text">
           <string>编码帧率</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">QLabel{color:#42464E; background-color:transparent;}</string>
          </property>
          <property name="text">
           <string>编码偏好</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_e_w">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>640</number>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QComboBox" name="comboBox_e_preference">
          <item>
           <property name="text">
            <string>Disabled(无偏好。不降低帧率和分辨率。)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Framerate(（默认值）帧率优先。分辨率不变。)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Quality(分辨率优先。)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Balance(平衡帧率与分辨率。)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="9">
         <widget class="QSpinBox" name="spinBox_e_fps">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">QLabel{color:#42464E; background-color:transparent;}</string>
          </property>
          <property name="text">
           <string>渲染模式</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_c_render">
          <item>
           <property name="text">
            <string>Hidden</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fit</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fill</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_13">
          <property name="styleSheet">
           <string notr="true">QLabel{color:#42464E; background-color:transparent;}</string>
          </property>
          <property name="text">
           <string>镜像效果</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox_morror">
          <item>
           <property name="text">
            <string>None(本地预览和编码传输时均无镜像效果)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Render(本地预览时有镜像效果，编码传输时无镜像效果)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RenderAndEncoder(本地预览和编码传输时均有镜像效果)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_12">
          <property name="styleSheet">
           <string notr="true">QLabel{color:#42464E; background-color:transparent;}</string>
          </property>
          <property name="text">
           <string>旋转角度</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="comboBox_rotation">
          <item>
           <property name="text">
            <string>0(不旋转)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>90(顺时针旋转 90 度)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>180(顺时针旋转 180 度)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>270(顺时针旋转 270 度)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="checkBox_water">
          <property name="text">
           <string>增加水印</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>远端视频设置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_22">
        <property name="styleSheet">
         <string notr="true">QLabel{color:#42464E; background-color:transparent;}</string>
        </property>
        <property name="text">
         <string>渲染模式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_remote_render">
        <property name="minimumSize">
         <size>
          <width>216</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Hidden</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fill</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UserWidget</class>
   <extends>QWidget</extends>
   <header>UserWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
